src/Model.php

```diff
@@ -727,7 +716,7 @@ public function getFields($filter = null)
      *
      * @return $this
      */
-    public function set($field, $value = null)
+    public function set(string $field, $value = null)
```

Comment:
No, no, no. `set(array)` is very handy and used widely.

Reply:
from Skype:

![image](https://user-images.githubusercontent.com/2228672/87336466-2fe0da80-c542-11ea-9499-e5a493aa48a4.png)


![image](https://user-images.githubusercontent.com/2228672/87336488-396a4280-c542-11ea-8746-1bbc14400f86.png)


![image](https://user-images.githubusercontent.com/2228672/87336511-40915080-c542-11ea-8314-460c74df5d56.png)